tasks/main.yml

```diff
@@ -34,7 +34,8 @@
     {{pki_cfssl_root}}/cfssl gencert
     -ca {{pki_dir}}/ca.pem
     -ca-key {{pki_dir}}/ca-key.pem
-    -hostname {{ ([item.cname] + (['localhost', '127.0.0.1'] if item.include_localhost is defined and item.include_localhost else []) + (item.sans|default([])) + (item.altips|default([]))) | join(",") }} csr.json 
+    -cn {{item.cname}}
+    -hostname {{ ([item.cname] + (['localhost', '127.0.0.1'] if item.include_localhost is defined and item.include_localhost else []) + (item.sans|default([])) + (item.altips|default([]))) | join(",") }} csr.json
```

Comment:
RFC 5280 says it can be in either or both, so having it in there still is fine.

Source: http://stackoverflow.com/questions/5935369/ssl-how-do-common-names-cn-and-subject-alternative-names-san-work-together